GitHub/1/README.md

```diff
@@ -7,12 +7,12 @@
 ## Challenge
 - Fork devopsloft/training repo
 - Clone it
-- Checkout to a branch
-- Implement a meaningful change to this challenge
+- Checkout to a feature branch
+- Pick up an issue and implement it
 - Submit a PR
 - Pass a code review
 - Verify your PR is merged
-- Update your local and origin repo with the merged changes
+- Sync your local and origin repos with upstream
 
```

Comment:
One last addition.
After the last sync - delete feature branch :)

Reply:
ack